spec/features/bulk_updates/bulk_updates_search_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+describe 'Searching collections to bulk update', reset_provider: true do
+  before(:all) do
+    # Create a few collections with unique attributes that we can search for
+    2.times { |i| publish_draft(version: "nasa.001#{i}") }
+    3.times { |i| publish_draft(short_name: "nasa.002#{i}") }
```

Comment:
I just meant that you could have used some searches similar to what is done in `search_collections_result_spec`. But I think the methods that we have to publish drafts and stipulate certain attributes is much improved since then.